flow-report/test/App-test.tsx

```diff
@@ -7,11 +7,10 @@
 import fs from 'fs';
 import {App} from '../App';
 import {render} from '@testing-library/preact';
-import {LH_ROOT} from '../../root';
 
 const flowResult = JSON.parse(
   fs.readFileSync(
-    `${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/reports/sample-lhrs.json`,
+    `${__dirname}/../../lighthouse-core/test/fixtures/fraggle-rock/reports/sample-lhrs.json`,
```

Comment:
similar to https://github.com/GoogleChrome/lighthouse/pull/12940#discussion_r692309054, `root.js` isn't accessible without importing all of core, so this works for now.

I'm also kind of confused on where `tsx` files live on the commonjs <-> esmodules spectrum and what works in them and what doesn't. Does it just depend on the bundler?